src/lib/run_environment.ml

```diff
@@ -28,10 +28,11 @@ module Tool = struct
       [@@@ocaml.warning "-11"]
       type t = [
         | `Bwa of [ `V_0_7_10 ]
-        | `Custom of string * string
+        (* A tool that is manually installed by retrieving source or binary.*)
```

Comment:
`` `Bwa _`` was the first one there, meant as an example. Many things in `` `Custom`` were meant to be upgraded to their own variant. But this never seemed a priority.